packages/flutter_tools/test/commands.shard/permeable/create_test.dart

```diff
@@ -2567,11 +2615,27 @@ Future<void> _analyzeProject(String workingDir) async {
     args,
     workingDirectory: workingDir,
   );
-  if (exec.exitCode != 0) {
-    print(exec.stdout);
-    print(exec.stderr);
+  if (expectedFailures.isEmpty) {
+    if (exec.exitCode != 0) {
+      print(exec.stdout);
+      print(exec.stderr);
+    }
+    expect(exec.exitCode, 0);
+    return;
   }
-  expect(exec.exitCode, 0);
+  expect(exec.exitCode, isNot(0));
+  String lineParser(String line) {
+    print('#$line#');
```

Comment:
was this a temporary debugging line that leaked in?

Reply:
Oh, yeah. Will send PR to remove.

Reply:
https://github.com/flutter/flutter/pull/91317